testing/trino-benchto-benchmarks/src/test/java/io/trino/sql/planner/TestTpchUnPartitionedCostBasedPlan.java

```diff
@@ -14,21 +14,16 @@
 
 package io.trino.sql.planner;
 
-import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
-import io.trino.plugin.tpch.TpchMetadata;
 
-import java.util.stream.IntStream;
-import java.util.stream.Stream;
-
-import static java.lang.String.format;
+import java.util.Map;
 
 /**
  * This class tests cost-based optimization rules related to joins. It contains unmodified TPCH queries.
  * This class is using Hive connector with mocked in memory thrift metastore with un-partitioned TPCH tables.
  */
-public class TestTpchCostBasedPlan
-        extends AbstractCostBasedPlanTest
+public class TestTpchUnPartitionedCostBasedPlan
```

Comment:
I don't think this needs a rename given that the partitioned case has been named explicitly